.github/workflows/ci.yml

```
@@ -361,6 +361,34 @@ jobs:
             ./build_tools/cmake/ctest_all.sh \
             "${BUILD_DIR}"
 
+  build_test_runtime_arm64:
+    needs: setup
+    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_test_runtime_arm64')
+    runs-on: <TODO>
+    env:
+      BUILD_DIR: build-runtime
```

nit: Maybe use a distinguishable directory name 
```suggestion
      BUILD_DIR: build-runtime-arm64
```

---

Will fix